Dockerfile.langflow

```
@@ -1,21 +1,55 @@
-# syntax=docker/dockerfile:1.4
-FROM langflowai/langflow:1.9.0
+w# syntax=docker/dockerfile:1.4
```

Hadolint (2.14.0)

[error] unexpected 'w'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)